### ISSUE TITLE
Detected as a virus.

### ISSUE THREAD

**Issue by peternrdstrm** [AndroidAppz/LuckyPatcher#8]
Alright, teenager tech entusiast here. 
As a youngling, I LOVED LP, it was one of my favorite apps. The thing is, that everywhere I look, it gives me a fake, that's full of viruses,shown by a virustotal scan. 
So I felt super happy, to find it on GitHub, only to see, that it's also detected, by 8 engines. (Worst was like 30, so still better I guess)
So what is wrong? I assume, that if it's on GitHub, it isn't malware, since everyone could see it in the published code. Or is it actully harmfull?

**Comment by PreciousWarrior** [comment]
Dude the app is closed source, in the `com.chelpus` source code there are some functions that couldn't be decompiled, which means the person maintaining the github repo isn't the actual author, they just took the APK and decompiled it and put it up on github. But most of it didn't get decompiled properly which happens with most apps that I tried. It probably has a virus or spyware or something of the sorts because the website and project looks fishy as hell and it isn't open source, NEVER grant a closed-source application root permissions, since you don't know what it could do. (unless its a test phone or something, I might install it on an old phone and use mitmproxy to see what requests the app is making). This news is pretty sad to me because I wanted some code that can programmatically disable APK signature verification without having to save data, uninstall, reinstall, and recover data. But oh well. BTW if anybody has an idea how to disable signature verification programmatically please do tell me.

**Comment by PreciousWarrior** [comment]
Also you sound like Anakin Skywalker saying youngling, very sus indeed.

**Comment by Bluscream** [comment]
yeah i have no affiliation with the app itself, im honestly surprised it wasn't dmca'ed by now. And yes the repo hasnt been updated since my bit for it broke and i cant be bothered to fix it. However im still using a more recent version of it on all my devices and i never had any issues that i can link to it
